.yarnrc.yml

```diff
@@ -1 +0,0 @@
-checksumBehavior: 'update'
```

Comment:
Yarn.lock contains an integrity checksum for each package, so that if the package is updated without a version change it will fail to install.

`checksumBehavior` dictates what happens when an installed package fails the integrity check. We want the integrity check to throw errors! Updating the checksum to match the new one is the opposite of what we want.

Strangely you cannot use `.yarnrc.yml` for CLI arguments like you can `.yarnrc` so I just deleted the file entirely.